docs/altos.md

```diff
@@ -18,9 +18,9 @@ Pros:
 
 Cons: 
 * Audio driver is paid (10 USD) for Skylake (6th generation) and newer devices
-* Thunderbolt driver is paid (20 USD) for certain generations
+* **Certain** devices with Thunderbolt ports will need to purchase an additional driver ($20 USD)
```

Comment:
All devices with thunderbolt ports are paid.

Maybe word this in the same way as the `Audio driver is paid (10 USD) for Skylake (6th generation) and newer devices` line. The thunderbolt driver is needed on tigerlake (11th gen) and up if I'm not mistaken

Reply:
Keep in mind not all of them have tbt, some are usb4 which also needs that driver.

Reply:
@ethanaobrien @death7654 

Maybe reword to `USB4/Thunderbolt driver is paid (20 USD) for TigerLake (11xxx) and newer`